bagitobjecttransfer/recordtransfer/settings.py

```diff
@@ -96,3 +96,5 @@
     cast=AcceptedFileTypes(),
     default="Archive:zip|Audio:mp3,wav,flac|Document:docx,odt,pdf,txt,html|Image:jpg,jpeg,png,gif|Spreadsheet:xlsx,csv|Video:mkv,mp4",
 )
+
+MODE = config("MODE", default="DEFAULT_MODE")
```

Comment:
MODE can be removed - this is one place we missed it

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/248/commits/ee359aacf3ca6f2c42350fbfcbe9de5eb549d3c0